app/views/jobseekers/job_applications/build/ask_for_support.html.slim

```diff
@@ -24,3 +24,6 @@
 
       = f.govuk_submit t("buttons.continue") do
         = f.govuk_submit t("buttons.save_as_draft"), secondary: true
+
+  .govuk-grid-column-one-third
+    = render(Shared::ProcessStepsComponent.new(process: job_application.vacancy, service: process_steps, title: t("jobseekers.job_applications.process_title")))
```

Comment:
As you've referenced `process_steps` twice in this page, perhaps it should be assigned just once in the controller, using:

```ruby
def process_steps
  @process_steps ||= ProcessSteps.etc.etc.etc
end
```

Reply:
goes against the grain of comments i had on the PR previously for the component??

Reply:
Oh does it? That's fine, mine is a very very minor optimization